pkg/features/kube_features.go

```diff
@@ -1054,7 +1054,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DisableKubeletCloudCredentialProviders: {Default: true, PreRelease: featuregate.Beta},
 
-	DisableNodeKubeProxyVersion: {Default: false, PreRelease: featuregate.Alpha},
+	DisableNodeKubeProxyVersion: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
there's a comment above, where `DisableNodeKubeProxyVersion` is defined, that needs to be updated to indicate the version where it became beta